### ISSUE TITLE
Improvement: add test location to jest call

### ISSUE THREAD

**Issue by sinnedh** [firsttris/vscode-jest-runner#13]
To improve the execution speed and ignore errors in other files one could add the path of the current test file to the jest call.

Jest only looks in this particular file and does not search for tests in all files.

E.g.
``` js
// Now:
jest -t "test name pattern"

// Better use:
jest -t "test name pattern" path/to/file.test.js
```

**Comment by firsttris** [comment]
this feature is now implemented with https://github.com/firsttris/vscode-jest-runner/pull/11

**Comment by firsttris** [comment]
included in version 0.2.5
